subprojects/android-test/test-instrumentation-runner/src/main/kotlin/com/avito/android/runner/Context.kt

```diff
@@ -0,0 +1,5 @@
+package com.avito.android.runner
+
+interface Context {
```

Comment:
What is the responsibility of this abstraction?
What will it know besides errors reporter?

Reply:
It is a service locator for the future. I consider to use it for holding all needed dependencies